src/test/java/com/fontys/api/service/UserServiceTest.java

```diff
@@ -0,0 +1,6 @@
+package com.fontys.api.service;
+
+public class UserServiceTest
```

Comment:
I think you can remove this, as it is empty.
This will be (re)created when User gets implemented